app/src/main/java/mx/yellowme/youst/challenges/ChallengeWithListActivity.kt

```diff
@@ -1,4 +1,4 @@
-package mx.yellowme.youst.common.activities;
+package mx.yellowme.youst.challenges;
```

Comment:
You can drop the `;` 🙈

Reply:
Legacy Java, thanks! Searching and replacing 🙌